suites/pacific/rgw/tier-2_rgw_regression.yaml

```diff
@@ -721,9 +721,6 @@ tests:
         test-version: v2
         script-name: test_rgw_mfa.py
         config-file-name: test_rgw_mfa_multipart.yaml
-        extra-pkgs:
-          8:
-            - https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/o/oathtool-2.6.2-3.el8.x86_64.rpm
```

Comment:
@anrao19 
you are removing it here because its enough to install the repo only for the first test in a suite right?

Reply:
Yes @hmaheswa